core/scratch_block_comment.js

```diff
@@ -0,0 +1,404 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2011 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Object representing a code comment.
+ * @author kchadha@scratch.mit.edu (Karishma Chadha)
+ */
+'use strict';
+
+goog.provide('Blockly.ScratchBlockComment');
+
+goog.require('Blockly.Comment');
+goog.require('Blockly.ScratchBubble');
```

Comment:
Alphabetical order, and an extra space between the Blockly requires and the goog requires.